libs/cms/src/lib/search/importers/article.service.ts

```diff
@@ -111,11 +111,19 @@ export class ArticleSyncService implements CmsSyncProvider<IArticle> {
           }
         }
 
-        if (!isCircular(processedEntry)) {
-          processedEntries.push(processedEntry)
-        } else {
-          logger.warn('Circular reference found in article', {
-            id: entry.sys.id,
+        try {
+          const mappedEntry = mapArticle(processedEntry)
+          if (!isCircular(mappedEntry)) {
+            processedEntries.push(processedEntry)
+          } else {
+            logger.warn('Circular reference found in article', {
+              id: entry?.sys?.id,
+            })
+          }
+        } catch (error) {
+          logger.warn('Failed to map article', {
```

Comment:
Consider logging this as error. Same for the `Circular reference found in article` above.
Apparently, there are many places like this. We should discuss what we should consider as error vs. warning, keeping in mind that we want to know when some data is not being synced/updated, as there is highly likely that someone is making changes in Contentful and waiting for these changes to be appear on the web.

Reply:
As per our discussions on Slack, we will not be logging these as errors, at least for now. Errors will call the operations team to action, which we don't want to do until we're confident that the indexer will not throw "too many" of those. In the meantime, we'll keep an eye on the logged warnings while we stabilize the indexer code.